damafeez/github-users
# Ch fastlane integration 156005300

**What does this PR do?**
This PR accesses the GitHub API (Finishes [story](https://www.pivotaltracker.com/n/projects/2155295/stories/156005300))

**Description of Task to be completed?**

- fastlane init
- add slack url to fastfile
- add add generated fastlane files to .gitignore
- build APK
- upload APK to converge-codelab-qa Slack channel
- add sentitive keys to environment variables

**How should this PR be manually tested**
- clone the repo
- run `fastlane slackbuild` from terminal
- observe Slack thread for messages and APK upload

**Any background context you want to provide?**
N/A

**What are the relevant pivotal tracker stories?**
#156005300

**Screenshots (if appropriate)**
![image](https://user-images.githubusercontent.com/26779819/50759171-80e48400-1264-11e9-9f92-a8ef8a4b8d3c.png)


**Questions:**
None
